Update WCA Documents

viroulep opened the pull request:
It updates the `update_docs` scripts to match the new documents tree introduced by thewca/wca-documents#120, and uses the pdf generated there as the competitor tutorial.

I also introduced a new "WQAC Educational Documents" page, but that's just to show how it can be done, we need to check with the WQAC what they actually want.

We should merge this just when thewca/wca-documents#120 is merged (and when the WQAC is happy with it obviously).

suushiemaniac commented on WcaOnRails/config/routes.rb:
```diff
@@ -145,7 +146,7 @@
   get 'wca-workbook-assistant-versions' => 'static_pages#wca_workbook_assistant_versions'
   get 'organizer-guidelines' => 'static_pages#organizer_guidelines'
-  get 'tutorial' => redirect('/files/WCA_Competition_Tutorial.pdf', status: 302)
+  get 'tutorial' => redirect('/edudoc/competitor-tutorial/tutorial.pdf', status: 302)
```
The Markdown version over in `wca-documents` is *NOT* feature-complete and a WIP. Please continue to use the static PDF from `files/` for the time being.

suushiemaniac commented on WcaOnRails/app/views/static_pages/education.html.erb:
```diff
@@ -0,0 +1,9 @@
+  <h1><%= yield(:title) %></h1>
+
+  <p>
+    Download the <%= link_to t('education.competitor_tutorial'), "/edudoc/competitor-tutorial/tutorial.pdf" %>.
```
Shouldn't "download the" also be translated?

viroulep replied:
Yes definitely, but I'm not even sure a page is needed so I'll loop in the WQAC first.

viroulep replied:
Done now that @leonopulos has replied in the email thread.

suushiemaniac commented on scripts/deploy.sh:
```diff
@@ -126,13 +126,14 @@ rebuild_regs() {
-  mv $tmp_dir $documents_folder
+  rm -rf $public_dir/documents
+  rm -rf $public_dir/edudoc
+  mv $tmp_dir/documents $tmp_dir/edudoc $public_dir
```
Wow, I never knew `mv` could take multiple arguments in the first position! :heart_eyes:

viroulep commented:
Screenshot: 
![wca-education](https://user-images.githubusercontent.com/1007485/68699937-53532000-0584-11ea-91e4-498627c7a049.png)

I improvised the text, but it can of course be adjusted.

DanielEgdal commented on WcaOnRails/config/locales/en.yml:
```diff
@@ -699,7 +699,7 @@ en:
       faq: "Frequently Asked Questions"
       teams: "Teams, Committees, and Councils"
-      tutorial: "Competition Tutorial"
+      tutorials: "Tutorials"
```
Maybe just rename this to education?

viroulep replied:
I've renamed that to "Education resources"

viroulep commented:
I've updated the PR to add a link to the drive containing the WMT templates, and removed the link to the Staff tutorial as it should appear in the staff panel.
![education_doc](https://user-images.githubusercontent.com/1007485/72188101-c7316d80-33f9-11ea-88eb-a4fa6680b722.png)

The page is currently as simple as it can be!

viroulep commented:
@thewca/wqac could you please review the content-related part of this PR?
It introduce the education page and links the staff/delegate crash course in the panel.

leonopulos commented:
The changes look like exactly what we wanted to me. You can consider this as a "Go!" from the WQAC. :)

viroulep commented:
@jonatanklosko: FYI I took this opportunity to remove the hacks we set up specifically to handle the crash course as a "hidden" post, which basically means removing the "world_readable" column and all code related to it.

Note to self: I need to remove the "hidden" post which holds the crash course after merging this.

viroulep commented on WcaOnRails/config/locales/en.yml:
```diff
@@ -762,7 +762,7 @@ en:
       forum: "Forum"
       teams: "Teams, Committees, and Councils"
-      tutorial: "Competition Tutorial"
+      education: "Education resources"
```
@moralsh @Ivan-Brigidano @AlbertoPdRF: would one of you mind providing me a suitable Spanish translation for that navigation entry please?

AlbertoPdRF replied:
`Recursos educativos` should be good enough.

viroulep replied:
Awesome thanks :)

viroulep commented:
I intend to merge this altogether with thewca/wca-documents#154 in the near future, it would be nice to have a review on the last commit which removes the crash course, if anyone is available (I'll probably do some WST stuff on next Wednesday).
